edweiss-firebase/functions/src/functions/memento/deleteDecks.ts

```diff
@@ -3,11 +3,12 @@ import { onAuthentifiedCall } from 'utils/firebase';
 import { CollectionOf } from 'utils/firestore';
 import { ok } from 'utils/status';
 
-export const deleteDeck = onAuthentifiedCall(Memento.Functions.deleteDeck, async (userId, args) => {
+export const deleteDecks = onAuthentifiedCall(Memento.Functions.deleteDecks, async (userId, args) => {
```

Comment:
Idem be careful of the new name of the function

Reply:
Got it!!